tests/TestCase/Installer/PluginInstallerTest.php

```diff
@@ -113,11 +130,41 @@ public function testGetInstallPath()
             )
         );
         $this->package->setAutoload($autoload);
-        $path = $this->installer->getInstallPath($this->package);
-        $this->assertEquals('plugins/Acme/Foo', $path);
+        $ns = $this->installer->primaryNamespace($this->package);
+        $this->assertEquals('Acme\Foo', $ns);
+    }
+
+    /**
+     * testUpdateConfigAddPath
+     *
+     * @return void
+     */
+    public function testUpdateConfigAddPath()
+    {
+        file_put_contents($this->path . '/config/plugins.php', '<?php return ["plugins" => ["Bake" => "/some/path"]];');
+
+        $this->installer->updateConfig('DebugKit', '/vendor/cakephp/debugkit');
+        $contents = file_get_contents($this->path . '/config/plugins.php');
+        $this->assertContains('<?php', $contents);
+        $this->assertContains("'DebugKit' => '/vendor/cakephp/debugkit/'", $contents);
+        $this->assertContains("'Bake' => '/some/path'", $contents);
+    }
```

Comment:
Oh, duh. I don't think it's necessary; not particularly fond of sequential strpos checks though.